src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -31,18 +32,23 @@
 	<Header back={route === 'settings'} />
 
 	<AuthGuard>
-		<Hero
-			usdTotal={route === 'tokens'}
-			summary={route === 'transactions'}
-			actions={route !== 'settings'}
-			back={route === 'transactions'}
-		/>
-
-		<main class="pt-8">
+		<MainLayout>
+			<div class="" slot="menu">
+				<!-- TODO: add menu-->
+			</div>
+
+			{#if route !== 'settings'}
+				<Hero
+					usdTotal={route === 'tokens'}
+					summary={route === 'transactions'}
+					back={route === 'transactions'}
+				/>
+			{/if}
+
 			<LoadersGuard>
 				<slot />
 			</LoadersGuard>
-		</main>
```

Comment:
There is no more main do I get it right?

Reply:
Well, `main` is inside `MainLayout`, do you want to separate them?

Reply:
Ah, my bad. It's good.